services/QuillLMS/app/lib/utils/numeric.rb

```diff
@@ -2,7 +2,7 @@
 
 module Utils::Numeric
   def self.to_human_string(numeric)
-    raise ArgumentError, "Not a numeric" unless numeric.is_a?(Numeric)
+    return numeric.to_s unless numeric.is_a?(Numeric)
```

Comment:
If something can be converted to a number, like these values in Redis, don't we actually want the utility of the `Numeric` conversion here?  Like, if we pull `"5700000"` out of Redis, don't we want this to actually return "6 million" or whatever?

Reply:
Yes, that's a nice enhancement; I've added it. I leveraged the fact the `to_f` will never raise an exception based on it's input. 

`asdf.to_f => 0.0`